File: messageBus/EmpujeComunitario.MessageFlow.Common/Model/MessagesRabbitMQ/VolunteerAdhesionModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmpujeComunitario.MessageFlow.Common.Model.MessagesRabbitMQ
{
    public class VolunteerAdhesionModel
    {
        public Guid EventId { get; set; }
        public Guid OrgId { get; set; }
        public string VolunteerId { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
    }
}

File: server/src/main/java/com/grpc/empuje_comunitario/domain/messages/VolunteerAdhesionModel.kt
package com.grpc.empuje_comunitario.domain.messages

data class VolunteerAdhesionModel(
    // ID del evento al que se adhiere
    val eventId: String,
    // ID Organización (la tuya)
    val orgId: String,
    // ID Voluntario
    val volunteerId: String,
    // Nombre del voluntario
    val name: String,
    // Apellido del voluntario
    val lastName: String,
    // Teléfono
    val phone: String? = null,
    //Email
    val email: String
)
